tests/unit/render_resource/simple_manifest_result.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   template:
     spec:
       containers:
-      - image: metacontroller/metacontroller:v0.3.0
+      - image: sample/metacontroller:tag
```

Comment:
Is this intentional?

Reply:
Yes.  I wanted it to be obviously not correct just so nobody thought it needed to be bumped in future version bumps.  It only gets used in a mock, not as a real resource

Reply:
+1, can you add a comment saying this is just for mocking things? It'll be more clear.